src/Ord.ts

```diff
@@ -226,6 +227,16 @@ export function getSemigroup<A = never>(): Semigroup<Ord<A>> {
   }
 }
 
+/**
+ * @since 2.3.1
+ */
+export function getMonoid<A>(): Monoid<Ord<A>> {
+  return {
+    empty: fromCompare((a1, a2) => 0),
```

Comment:
We can define `empty` once and for all

```ts
const empty = fromCompare(() => 0)

/**
 * @since 2.4.0
 */
export function getMonoid<A>(): Monoid<Ord<A>> {
  return {
    empty,
    concat: getSemigroup<A>().concat
  }
}
```